### ISSUE TITLE
± sign in the CSV file

### ISSUE THREAD

**Issue by theMyth721** [devstronomy/nasa-data-scraper#3]
Hi 

The CSV file for the Moons has this ± sign in between numbers and its extremely difficult to upload.

I am trying to upload into Postgres on my Mac with Table plus, 

The JSON and SQL is a bit difficult, because I have my own schema and structure, and it would be very tedious for me to change manually

May i ask what is the point of keeping numbers in this format ?

And would there be a way to change this ?

Thanks

**Comment by mkrauskopf** [comment]
Yes. You are probably right. Since we have stripped the `±` sign from other formats (JSON and SQL) it should not be part of the final CSV file as well. So I would consider this to be a bug.

I do not use the data myself actively, nor do I work on the project anymore. Probably the easiest would be to run a `sed` on the current `satellites.csv` and strip the `±<number>` parts.

I might have time next week, but no promise. So the patch/PR is welcomed.

**Comment by theMyth721** [comment]
Hey, Thank you so much for getting back to me, So appreciated, I can try doing that myself. 
I am not sure what sed is but i just did man sed, and figured it's a stream editor.
If you would tell me the command then i can do this that would be great, otherwise i am trying to figure it out
Thanks

**Comment by mkrauskopf** [comment]
`sed` was just an example. What I mean is to just strip the `±<number>` parts. Manually, with the `sed`, with Vim macro, with your IDE search-and-replace functionality, ... It really does not matter much.

**Comment by theMyth721** [comment]
Thank you, I will try and do it

**Comment by theMyth721** [comment]
I just want to make sure, if I strip the entire right side after the ±, I hope the number value will be the same ?

**Comment by theMyth721** [comment]
I just got it to work in VSCode with this Regex: /±\d*\.?\d*/

I am still new to Github and Software Engineering, but Im happy to make the change for you in your file. I am not sure how to do Pull requests properly, but I have updated the file, and I am going to learn this too.

<img width="460" alt="Screenshot 2023-11-15 at 20 05 23" src="https://github.com/devstronomy/nasa-data-scraper/assets/88504671/e9ee8337-20ce-4421-8e9c-41028838ccb0">

**Comment by mkrauskopf** [comment]
It's worth investing in the skill. You can start with the official documentation: [Pull requests documentation - GitHub Docs](https://docs.github.com/en/pull-requests)

You can use this issue as your real training ;)
